docs/javascript/browser-apis-and-jquery.md

```diff
@@ -956,3 +956,122 @@ setTimeout(removeEvent, 5000);
 ![offMethod example](https://github.com/codewizardshq/docs/blob/main/docs/assets/browser-apis-and-jquery/offmethod.gif?raw=true){ width="100%" }
 <figcaption></figcaption>
 </figure>
+
+## Getting and Setting attributes
+
+### The `getAttribute()` function allows you to access the value of an element's attribute
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Home</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+<div>
+  <div class="object" id="one"></div>
+  <img class="object" id="two">
+
+</div>
+<script>
+
+  var elementAttribute = document.querySelector('div').getAttribute("id");
+  console.log("Attribute:", elementAttribute);
+
+</script>
+</html>
+```
+
+Example Output:
+
+```javascript
+Attribute: one;
+```
+
+### The `setAttribute()` function allows you to set or update the attribute of an element
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Home</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+<div>
+
+  <div class="object" id="one"></div>
+  <img class="object" id="two">
+
+</div>
+<script>
+
+  var insertImage = document.getElementById('two');
+  insertImage.setAttribute('src', 'dog.png');
+
+</script>
+</html>
+
+```
+
+Example Output:
+
+<figure markdown>
+![setAttribute example](https://github.com/codewizardshq/docs/blob/main/docs/assets/browser-apis-and-jquery/setAttribute.png?raw=true){ width="100%" }
+<figcaption></figcaption>
+</figure>
+
+### jQuery's `attr()` function allows you access or add an attribute to an element
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Home</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+<div>
+
+  <img class="object" id="one">
+  <img class="object" id="two">
+
+</div>
+<script>
+
+   function clickedPicture(event) {
+      var clicked = event.currentTarget;
```

Comment:
Use `event.target` anytime you need to get the target of an event. I have no idea why CWHQ was using `currentTarget`, but that's more typing than necessary and we don't teach event delegation here so it's a useless use of that property. jQuery handles the delegation for us in this instance to the proper target.